src/components/MatchItem/MatchInfo.tsx

```diff
@@ -0,0 +1,73 @@
+import { Typography, Box } from '@mui/material'
+import dayjs from 'dayjs'
+
+interface Props {
+  boardgameName: string
+  date: string
+  duration: number
+}
+
+export const MatchInfo = ({ boardgameName, date, duration }: Props) => {
+  const [weekDay, month, day] = dayjs(date).format('ddd, MMM, D').split(',')
```

Comment:
Splitting them like this will leave a white-space before the month and the day. Can we just put a white-space between `ddd MMM D` and then splitting like this? `split(' ')`

Reply:
Yes, we can! Done.